src/frontend/src/lib/components/dapps/DappsCarousel.svelte

```diff
@@ -30,6 +36,18 @@
 		if (idx !== -1) {
 			carousel.removeSlide(idx);
 		}
+
+		if (isNullish($authIdentity) || isNullish($userProfileStore)) {
```

Comment:
For `isNullish($authIdentity)` we can use `await nullishSignOut();`.

Reply:
Good one!!! Included!